MODULE.bazel

```diff
@@ -0,0 +1,23 @@
+module(
+    name = "rules_rust",
+    version = "0.9.0",
+)
```

Comment:
rules_go put the following in their MODULE.bazel. I think we should add something similar to rules_rust:

`print("WARNING: The rules_go Bazel module is still highly experimental and subject to change at any time. Only use it to try out bzlmod for now.")`

https://github.com/bazelbuild/bazel-central-registry/blob/main/modules/rules_go/0.33.0/MODULE.bazel#L7